components/header/Navbar.tsx

```diff
@@ -16,30 +16,40 @@ import { navbarHeight } from "./constants.ts";
 import { Buttons, Logo } from "../../components/header/Header.tsx";
 
 // Make it sure to render it on the server only. DO NOT render it on an island
-function Navbar(
-  { items, searchbar, logo, buttons, logoPosition = "left", device }: {
-    items: SiteNavigationElement[];
-    searchbar?: SearchbarProps;
-    logo?: Logo;
-    buttons?: Buttons;
-    logoPosition?: "left" | "center";
-    device: "mobile" | "desktop" | "tablet";
-  },
-) {
+function Navbar({
+  items,
+  searchbar,
+  logo,
+  buttons,
+  logoPosition = "left",
+  device,
+}: {
+  items: SiteNavigationElement[];
+  searchbar?: SearchbarProps;
+  logo?: Logo;
+  buttons?: Buttons;
+  logoPosition?: "left" | "center";
+  device: "mobile" | "desktop" | "tablet";
+}) {
   const platform = usePlatform();
 
   // Mobile header
-  if (device === "mobile") {
+  if (device === "mobile" || device === "tablet") {
     return (
```

Comment:
A condicional pode ser transformada em ternário para o retorno do componente:
```suggestion

return (device === "mobile" || device === "tablet") && (
```